### ISSUE TITLE
iOS - preview blurred and no scan on Pro models (with 3 cameras)

### ISSUE THREAD

**Issue by jishinhv** [Chaintech-Network/QRKitComposeMultiplatform#3]
> QRKit version - ~~1.0.2~~ ~~1.0.3~~ 1.0.6
> XCode version - 15.4
> Kotlin version - 1.9.23
> AGP - 8.2.2

The library works as intended on android, but running it on iOS has been a challenge. 

### Issues noticed
1. In the last week only once was when the scan worked on iOS, haven't then been able to replicate the same. How can I debug what could be going wrong? I tried running the demo app in the repo but the build never succeeds. - ~~**RESOLVED** after updating to `1.0.3`~~ This issue is still happening, scans are not working most times. Tested on IPhone SE2 and iPhone Xr
2. On the iPhone 15 pro, the preview is always blurred and it does not focus when held up towards the QR. 
3. Preview is stuck post scan, which I believe has been already reported [here](https://github.com/Chaintech-Network/QRKitComposeMultiplatform/issues/2).

Can you guide on what steps can be taken to either debug or fix this please? Let me know if you need any more info.

**Comment by ctn-mobile-innovation** [comment]
Thanks for reporting! Let us check and get back to you

**Comment by jishinhv** [comment]
@ctn-mobile-innovation we would be happy to contribute for the blur issue as this is something we've fixed on native for iOS. Is the library open sourced elsewhere or is it closed? Could not find it in this repo.

**Comment by jishinhv** [comment]
@ctn-mobile-innovation any updates here?

**Comment by ctn-mobile-innovation** [comment]
We have worked on fix! will publish an update this week, and yes we will be happy if you can contribute!

**Comment by jishinhv** [comment]
Was the fix released @ctn-mobile-innovation ? Any ETAs?

**Comment by ctn-mobile-innovation** [comment]
@jishinhv Please check in new version and let us know!

**Comment by jishinhv** [comment]
@ctn-mobile-innovation The app is frozen on launch and I get the following error when running on iOS with the latest version `1.0.7` - 
```
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '*** -[AVCaptureMetadataOutput setMetadataObjectTypes:] Unsupported type found (catBody) - use -availableMetadataObjectTypes'
*** First throw call stack:
(0x19ba06418 0x198d6de88 0x1ba273720 0x1069cfb1c 0x1069d3a34 0x1069d4a78 0x106150740 0x1066beff4 0x1066c06b0 0x10614f1d8 0x1062aebc8 0x1063b50ac 0x1062a45cc 0x10629f530 0x1062a005c 0x1063b47f8 0x1062c5268 0x1062c97bc 0x1062c9a04 0x10614f1d8 0x106270ad4 0x1062713a8 0x1066818fc 0x10675ac84 0x1066e5cd8 0x1054a4a60 0x10673be64 0x10676593c 0x10673277c 0x106735068 0x10673716c 0x10673723c 0x106150740 0x106737f84 0x106738038 0x19d509e90 0x19d5081fc 0x19d507d94 0x19e2141bc 0x19e213e0c 0x19e214ca0 0x19b9d928c 0x19b9d9220 0x19b9d6d5c 0x19b9d5f04 0x19b9d5714 0x1e74d11c4 0x19e549564 0x19e5f7530 0x1a011fefc 0x1a00fa538 0x1a00fd398 0x1049d8474 0x1049d8520 0x1c157fcd0)
libc++abi: terminating due to uncaught exception of type NSException
```

**Comment by jishinhv** [comment]
Hey @ctn-mobile-innovation, We've not been able to confirm the fix released with `1.0.7` because of the above reported crash and would really appreciate if you could help us out here with a quick resolution.

**Comment by jishinhv** [comment]
Raised a new issue for the crash  - https://github.com/Chaintech-Network/QRKitComposeMultiplatform/issues/7

**Comment by ctn-mobile-innovation** [comment]
Please check in Latest version 1.0.8

**Comment by jishinhv** [comment]
This works. Thanks @ctn-mobile-innovation 🙌
